tsconfig.build.json

```diff
@@ -3,8 +3,8 @@
 	// Modified config to only compile .ts-files in the src dir
 	"compilerOptions": {
 		"noEmit": false,
-		"declaration": false
+		"declaration": true
 	},
-	"include": ["src/**/*.ts", "templates/**/*.ts"],
+	"include": ["src/**/*.ts", "templates/**/*.ts", "index.ts", "core.ts"],
```

Comment:
This causes the build dir to include a folder `src` because the two files are outside the rootDir, which is pretty ugly.

I would either hand-code a .js file that just contains `module.exports = require("./build/index")` and a corresponding `.d.ts` file that looks like the one that is currently generated (minus the `src`) in the path.
Or we should look into using the new `exports` in `package.json`
https://nodejs.org/api/packages.html#packages_exports

Reply:
Try it with an older version. The build directory already contained a "src" directory before (reason for this is the templates that are not in the build directory). I would have preferred to not have that directory, but without moving all templates to the src directory, there is no easy solution to this.

Reply:
ahh ok then